package.json

```diff
@@ -1339,10 +1339,10 @@
     "uuid": "11.1.0",
     "vega": "^5.33.0",
     "vega-interpreter": "^1.0.4",
-    "vega-lite": "^5.5.0",
+    "vega-lite": "^5.21.0",
```

Comment:
Do you think there will be issues upgrading to `6.x`?

Reply:
I would go this step first, and let renovate upgrade to the major once we ensure this passes